src/NuGet.Core/NuGet.Frameworks/NuGetFrameworkFactory.cs

```diff
@@ -196,37 +196,63 @@ public static NuGetFramework ParseFolder(string folderName, IFrameworkNameProvid
                             || mappings.TryGetVersion(parts.Item2, out version))
                         {
                             var profileShort = parts.Item3;
-                            string profile = null;
-                            if (!mappings.TryGetProfile(framework, profileShort, out profile))
+                            
+                            if (version.Major >= 5
```

Comment:
why not use `IsNet5Era` property instead of this condition.

Reply:
IsNet5Era is on NuGetFramework type...which this routine is about to create, so we cannot use that property.